RadixWallet/Clients/CloudBackupClient/CloudBackupClient+Live.swift

```diff
@@ -156,11 +156,13 @@ extension CloudBackupClient {
 				try? userDefaults.removeLastCloudBackup(for: profile.id)
 			}
 
-			let needsBackUp = profile.appPreferences.security.isCloudProfileSyncEnabled && profile.header.isNonEmpty
+			let isSynced = profile.appPreferences.security.isCloudProfileSyncEnabled
```

Comment:
We could also abandon early if iCloud sync is disabled 
```Swift
guard profile.appPreferences.security.isCloudProfileSyncEnabled else { return }
```

Reply:
Currently we will still check if the profile has been claimed on another phone, but you could argue that we don't care about that if they have disabled iCloud sync. That would simplify things.

Reply:
True, that was recently changed, will simplify this now